administrator/components/com_templates/src/Model/TemplateModel.php

```diff
@@ -1985,4 +1989,161 @@ protected function checkFormat($ext)
 
 		return in_array(strtolower($ext), $this->allowedFormats);
 	}
+
+	/**
+	 * Method to get a list of all the files to edit in a template's media folder.
+	 *
+	 * @return  array  A nested array of relevant files.
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function getMediaFiles()
+	{
+		$result = array();
+		$template = $this->getTemplate();
+
+		if (!isset($template->xmldata))
+		{
+			$template->xmldata = TemplatesHelper::parseXMLTemplateFile($template->client_id === 0 ? JPATH_ROOT : JPATH_ROOT . '/administrator', $template->name);
+		}
+
+		if (!isset($template->xmldata->inheritable) || (isset($template->xmldata->parent) && $template->xmldata->parent === ''))
+		{
+			return [];
```

Comment:
reuse `$result` which is a blank array already.

```suggestion
			return $result;
```